packs/tfe_fdo_nomad/README.md

```diff
@@ -0,0 +1,161 @@
+| `tfe_object_storage_s3_use_instance_profile` | no       | `"false"`                                    | Whether to use the instance profile for authentication.                                                                                                                                                                                                                             |
+| `tfe_object_storage_s3_endpoint`             | yes      | `""`                                         | The endpoint of the S3 compatible object storage being used.                                                                                                                                                                                                                        |
+| `tfe_object_storage_s3_access_key_id`        | yes      | `""`                                         | The access key id value to be used to query the S3 object storage bucket.                                                                                                                                                                                                           |
+| `tfe_redis_host`                             | yes      | `""`                                         | The Redis host name being used.                                                                                                                                                                                                                                                     |
+| `tfe_redis_user`                             | no       | `""`                                         | Redis server user.                                                                                                                                                                                                                                                                  |
+| `tfe_redis_use_tls`                          | no       | `"false"`                                    | Indicates to use TLS to access Redis.                                                                                                                                                                                                                                               |
+| `tfe_redis_use_auth`                         | no       | `"false"`                                    | Indicates Redis server is configured to use TFE_REDIS_PASSWORD and TFE_REDIS_USER (optional) for authentication.                                                                                                                                                                    |
```

Comment:
```suggestion
| `tfe_redis_use_tls`                          | no       | `false`                                    | Indicates to use TLS to access Redis.                                                                                                                                                                                                                                               |
| `tfe_redis_use_auth`                         | no       | `false`                                    | Indicates Redis server is configured to use TFE_REDIS_PASSWORD and TFE_REDIS_USER (optional) for authentication.                                                                                                                                                                    |
```

Reply:
`variable "tfe_redis_use_tls" {
  description = "The tls settings for redis to be used by TFE"
  type        = string
  default     = "false"
}

variable "tfe_redis_use_auth" {
  description = "The auth settings to be used by redis"
  type        = string
  default     = "false"
}`

Both are strings.

Reply:
Should it be? If we're quoting them in the HCL2 anyways doesn't having it be a bool in the variables file improve type safety and prevent typos?

Reply:
Makes sense. Let me update this and run the tests.